backend/core/app_initializer_v2.py

```
@@ -69,26 +70,26 @@ def initialize_app_state() -> Tuple[Dict[str, Any], SmartIllustrationService, Ba
     # Check for admin-triggered refresh flag
     refresh_flag_file = backend_dir / ".refresh_required"
     if refresh_flag_file.exists():
-        logger.info("Admin refresh flag detected - forcing rebuild")
+        logger.info("🔄 Admin refresh flag detected - forcing rebuild")
         force_rebuild = True
         # Remove the flag file after processing
         try:
             refresh_flag_file.unlink()
-            logger.info("Admin refresh flag processed and removed")
+            logger.info("✅ Admin refresh flag processed and removed")
         except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)